source-mysql/backfill.go

```diff
@@ -50,6 +50,11 @@ func (db *mysqlDatabase) ScanTableChunk(ctx context.Context, info sqlcapture.Tab
 
 	// Process the results into `changeEvent` structs and return them
 	var events []sqlcapture.ChangeEvent
+	logrus.WithFields(logrus.Fields{
+		"schema": schema,
+		"table":  table,
+		"rows":   len(results.Values),
+	}).Debug("translating query rows to change events")
```

Comment:
Doesn't this log sensitive row data or it's not a concern here?

Reply:
No, it's just logging the length of the result set, not its contents (which would be a crapton of data we wouldn't want to log purely on pragmatic grounds, in addition to the privacy concerns).